### PR TITLE
A bunch of random functions

### PR DESCRIPTION
**Pull request by Reinmmar**
I just browsed ghidra and looked if a function exists

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -11856,7 +11856,7 @@ class MenuGameLayer : cocos2d::CCLayer {
 	void updateColors() = imac 0x5056e0, m1 0x45d2b8;
 
 	virtual void update(float) = win 0x3118e0, m1 0x45d330, imac 0x505750;
-	virtual bool init() = win 0x310ac0, m1 0x45c2d0, imac 0x5045b0;
+	virtual bool init() = win 0x310ac0, m1 0x45c2d0, imac 0x50;
```
Comment (dankmeme01) [inline bindings/2.206/GeometryDash.bro:11859]: what happened here

File: bindings/2.206/GeometryDash.bro [hunk 2]
```diff
@@ -1254,16 +1254,16 @@ class CCPartAnimSprite : cocos2d::CCSprite {
 	TodoReturn dirtify();
 	TodoReturn getSpriteForKey(char const*);
 	TodoReturn initWithAnimDesc(char const*, cocos2d::CCTexture2D*, bool);
-	void setColor(cocos2d::ccColor3B);
+	void setColor(cocos2d::ccColor3B*) = win 0x45940;
```
Comment (matcool) [inline bindings/2.206/GeometryDash.bro:1257]: :(

File: bindings/2.206/GeometryDash.bro (hunk 1 above)
Comment (Reinmmar) [reply bindings/2.206/GeometryDash.bro:11859]: Wait fuck, my bad

File: bindings/2.206/GeometryDash.bro (hunk 2 above)
Comment (Reinmmar) [reply bindings/2.206/GeometryDash.bro:1257]: I assume this is referring to me making it a pointer? Lemme change that back and make a new PR

**Pull request merged**